### ISSUE TITLE
Color text that has been edited since your last save

### ISSUE THREAD

**Issue by MysticMismagius** [HexFiend/HexFiend#306]
HxD does this, and it makes it much easier to double-check that you got everything when making multiple edits
![Screen_Shot_2020-12-13_at_2 16 53_PM](https://user-images.githubusercontent.com/78046059/109437311-49ac3080-79f2-11eb-8a0a-85e19213e930.png)

**Comment by gingerbeardman** [comment]
Yes, please. 010 Editor also does this and is one of the reasons I use it often.
